shared/src/hooks/web3Context.tsx

```diff
@@ -39,12 +48,18 @@ export const useAvaxWeb3Context = () => {
   return useContext(AvaxWeb3Context);
 };
 
+export const useAuroraWeb3Context = () => {
+  return useContext(AuroraWeb3Context);
+};
```

Comment:
I don't see anywhere where `useAvaxWeb3Context` and `useAuroraWeb3Context` is used, so we can remove it for now. The children components are all using `useWeb3Context` to access the provider directly

Reply:
Removed.